openlibrary/macros/TypeChanger.html

```diff
@@ -12,14 +12,14 @@
         <div class="label">
             <label for="type.key">$_("Page Type")</label>
             <span class="small gray">&nbsp;<a href="javascript:;"
-                aria-haspopup="true" aria-controls="typeSplain" class="slide-toggle">Huh</a>?</span>
+                aria-haspopup="true" aria-controls="typeSplain" class="slide-toggle">$_('Huh?')</a></span>
```

Comment:
This kind of informal content might cause problems for translators and isn't the best in English. Not really an issue for this PR, but I think reviewing some of this content should be a separate task.

Reply:
I agree. This "folksy" style permeates the UI, as does embedded styling to **emphasize** things that don't really need emphasis, and a plethora of different ways of saying the same thing. Even if we can't eliminate all the redundant code, we should adopt consistent, standardized terminology and phrasing to reduce the amount of text which needs to be translated and make it easier to translate.

There are definitely sections where I adopted a lower effort, imperfect internationalization as a tactic to make progress on the overall goal. Even things which have been internationalized can be improved and we should review the contents of the French `messages.po` file with an eye towards improving things in a way that will reduce the work for future translations.

Now is the time to do this, before we do a second or third language.